packages/components/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -73,6 +75,22 @@ export const useMenuTrigger = (childProps: MenuTriggerChildProps): MenuTriggerSt
     [childOnAccessibilityAction, setOpen],
   );
 
+  const accessibilityPositionInSet = React.useMemo(() => {
```

Comment:
I don;t think this needs to be memo'ed. Comparing primitives to determine if a component should rerender is generally safe. It's objects that need to be memoized.